core/test/functional/routes/api/authentication_spec.js

```diff
@@ -173,4 +178,39 @@ describe('Authentication API', function () {
                 done();
             });
     });
+
+    it('reset password', function (done) {
```

Comment:
we didn't had a successful reset password test. by adding this, i was able to verify that the brute key's are correct.